cross/docker-tvheadend/Makefile

```diff
@@ -0,0 +1,25 @@
+PKG_NAME = docker-tvheadend
+PKG_VERS = master
```

Comment:
@th0ma7 _master_ is not suitable for version, as it changes often, and therefore the checksum check fails (as in https://github.com/SynoCommunity/spksrc/actions/runs/2364987801)

Is it possible to use official releases like [42078692-ls128](https://github.com/linuxserver/docker-tvheadend/archive/refs/tags/42078692-ls128.tar.gz)?

Reply:
It's now the case in my new on-going PR #5277 as it now contains the new tv grabber  :)